src/styles/CompareCards.ts

```diff
@@ -62,6 +62,11 @@ export const CompareCardsStyles = (mode: string) => {
         '.edit-btn-wrapper': {
           display: 'flex',
           justifyContent: 'end',
+          $nest: {
+            button: {
+              color: isTrueLight ? Colors.LinkText : Colors.LinkTextDark,
+            },
+          },
```

Comment:
I haven't folllowed closely how this button is styled, but could it be easier to use `variant="text" color="primary"` on the button component? Then `color="primary"` should follow the palette automatically. If that's the same colors of course.

Reply:
Nice!